docs/source/examples/lora_finetune.rst

```diff
@@ -288,8 +290,8 @@ Let's run this experiment. We can also increase alpha (in general it is good pra
 
 .. code-block:: bash
 
-    tune run --nnodes 1 --nproc_per_node 2 lora_finetune_distributed --config lora_finetune_distributed \
-    lora_attn_modules='[q_proj, k_proj, v_proj, output_proj]' \
+    tune run --nnodes 1 --nproc_per_node 2 lora_finetune_distributed --config llama2/7B_lora \
+    lora_attn_modules=['q_proj','k_proj','v_proj','output_proj'] \
```

Comment:
Oh interesting, does this work for you? Whenever I do this override for checkpointer.checkpoint_files I need to set the list as '[]' i.e. a string for this to be correctly parsed

Reply:
Yeah the secret is the lack of spaces